src/vue-components/modals/editElementGeneral.vue

```diff
@@ -130,4 +277,31 @@
     .row, .srow {
         margin-top: 1em;
     }
+
+    .input-button {
+        position: absolute;
+        right: 0;
+        height: 100%;
+        line-height: initial;
+        margin: 0;
+        padding: 0 1em;
+        box-shadow: none;
+        background-color: transparent;
+        cursor: pointer;
+        z-index: 1;
+        pointer-events: auto;
+    }
+
+    .checkbox-label-small {
```

Comment:
the select for the languages + `show all languages` checkbox should also be moved to an extra component and used everywhere, to prevent doing the same things (like computing currently shown languages etc.) at so many places.
But only do it if you want - since it's probably some work to do it everywhere...